src/m365/spo/commands/applicationcustomizer/applicationcustomizer-get.spec.ts

```diff
@@ -0,0 +1,692 @@
+  });
+
+  it('humanize scope shows the scope odata value when is different than 2 and 3', () => {
+    const actual = (command as any)["humanizeScope"](1);
+    assert(actual === "1");
+  });
+
+  it('retrieves an application customizer by id', async () => {
+    sinon.stub(request, 'get').callsFake(async (opts) => {
+      if (opts.url === `${webUrl}/_api/Web/UserCustomActions(guid'14125658-a9bc-4ddf-9c75-1b5767c9a337')`) {
+        return {
+          "ClientSideComponentId": clientSideComponentId,
+          "ClientSideComponentProperties": "",
+          "CommandUIExtension": null,
+          "Description": null,
+          "Group": null,
+          "Id": id,
+          "ImageUrl": null,
+          "Location": "ClientSideExtension.ApplicationCustomizer",
+          "Name": title,
+          "RegistrationId": null,
+          "RegistrationType": 0,
+          "Rights": "{\"High\":0,\"Low\":0}",
+          "Scope": "3",
+          "ScriptBlock": null,
+          "ScriptSrc": null,
+          "Sequence": 0,
+          "Title": title,
+          "Url": null,
+          "VersionOfUserCustomAction": "16.0.1.0"
+        };
+      }
+
+      throw 'Invalid request';
+    });
+
+    await command.action(logger, {
+      options: {
+        id: id,
+        webUrl: webUrl,
+        scope: 'Web'
+      }
+    });
+
+    assert(loggerLogSpy.calledWith({
+      ClientSideComponentId: clientSideComponentId,
+      ClientSideComponentProperties: '',
+      CommandUIExtension: null,
+      Description: null,
+      Group: null,
+      Id: id,
+      ImageUrl: null,
+      Location: 'ClientSideExtension.ApplicationCustomizer',
+      Name: title,
+      RegistrationId: null,
+      RegistrationType: 0,
+      Rights: '"{\\"High\\":0,\\"Low\\":0}"',
+      Scope: '3',
+      ScriptBlock: null,
+      ScriptSrc: null,
+      Sequence: 0,
+      Title: title,
+      Url: null,
+      VersionOfUserCustomAction: '16.0.1.0'
+    }));
+  });
+
+  it('retrieves an application customizer by title', async () => {
+    sinon.stub(request, 'get').callsFake(async (opts) => {
+      if (opts.url === `${webUrl}/_api/Web/UserCustomActions?$filter=Title eq 'Some%20customizer'`) {
+        return {
+          value: [
+            {
+              "ClientSideComponentId": clientSideComponentId,
+              "ClientSideComponentProperties": "",
+              "CommandUIExtension": null,
+              "Description": null,
+              "Group": null,
+              "Id": id,
+              "ImageUrl": null,
+              "Location": "ClientSideExtension.ApplicationCustomizer",
+              "Name": title,
+              "RegistrationId": null,
+              "RegistrationType": 0,
+              "Rights": "{\"High\":0,\"Low\":0}",
+              "Scope": "1",
+              "ScriptBlock": null,
+              "ScriptSrc": null,
+              "Sequence": 0,
+              "Title": title,
+              "Url": null,
+              "VersionOfUserCustomAction": "16.0.1.0"
```

Comment:
since we are repeating the same mock a couple of times maybe we could extract it to some mock variable at the top of the test definition